### PR TITLE
Document Google OpenID Connect email attribute

### PR DESCRIPTION
**Pull request by ptman**
In the example config

### Pull Request Checklist

* [x] Pull request is based on the develop branch
* [x] Pull request includes a [changelog file](https://matrix-org.github.io/synapse/latest/development/contributing_guide.html#changelog). The entry should:
  - Be a short description of your change which makes sense to users. "Fixed a bug that prevented receiving messages from other servers." instead of "Moved X method from `EventStore` to `EventWorkerStore`.".
  - Use markdown where necessary, mostly for `code blocks`.
  - End with either a period (.) or an exclamation mark (!).
  - Start with a capital letter.
  - Feel free to credit yourself, by adding a sentence "Contributed by @github_username." or "Contributed by [Your Name]." to the end of the entry.
* [x] Pull request includes a [sign off](https://matrix-org.github.io/synapse/latest/development/contributing_guide.html#sign-off)
* [ ] [Code style](https://matrix-org.github.io/synapse/latest/code_style.html) is correct
  (run the [linters](https://matrix-org.github.io/synapse/latest/development/contributing_guide.html#run-the-linters))

### CODE REVIEW

**Comment by ptman** [comment]
This is a very show documentation change that I made using the GitHub web editor. Please let me know if all the boxes still need to be ticked

File: docs/openid.md [hunk 1]
```diff
@@ -336,11 +336,12 @@ oidc_providers:
        issuer: "https://accounts.google.com/"
        client_id: "your-client-id" # TO BE FILLED
        client_secret: "your-client-secret" # TO BE FILLED
-       scopes: ["openid", "profile"]
+       scopes: ["openid", "profile", "email"] # email is optional, read below
```
Comment (DMRobertson) [inline docs/openid.md:339]: I'm guessing this corresponds to the three scopes listed here?

https://developers.google.com/identity/protocols/oauth2/scopes#oauth2

Comment (DMRobertson) [reply docs/openid.md:339]: Sanity check: have you used this config yourself and confirmed that it works? (On matrix.org we are using the scope name `https://www.googleapis.com/auth/userinfo.email`.)

**Review by DMRobertson — CHANGES_REQUESTED** [review]
> This is a very short documentation change that I made using the GitHub web editor. Please let me know if all the boxes still need to be ticked

Many thanks for this. Per that comment with the tickboxes, could you please:
- add a changelog entry
- a comment with your sign-off

There's no need to run the linters for a documentation-only change like this.

**Comment by ptman** [comment]
>     * add a changelog entry
>     * a comment with your sign-off

How do you perform these steps using the GitHub web interface?

> There's no need to run the linters for a documentation-only change like this.

Thanks, that's what I thought.

**Comment by DMRobertson** [comment]
> How do you perform these steps using the GitHub web interface?

For adding a changelog entry, I think you want to go to [the fork and branch that github has created for you](https://github.com/ptman/synapse/tree/patch-1) and follow the instructions [here](https://docs.github.com/en/repositories/working-with-files/managing-files/creating-new-files).

For adding a sign-off, writing a comment on this PR with `Signed-off-by:` as per [the notes here](https://matrix-org.github.io/synapse/latest/development/contributing_guide.html#sign-off) is all that's needed.

**Comment by clokep** [comment]
> > ```
> > * add a changelog entry
> > * a comment with your sign-off
> > ```
> 
> How do you perform these steps using the GitHub web interface?

You can go to https://github.com/ptman/synapse/tree/patch-1/changelog.d and click the `+` in the top right and choose "Create new file".

For sign-off you can just add it manually via a comment on the PR.

**Comment by ptman** [comment]
Signed-off-by: Paul Tötterman <paul.totterman@iki.fi>

**Review by DMRobertson — APPROVED** [review]
Many thanks, SGTM!

**Pull request merged**